OpenRA.Game/ExternalMods.cs

```diff
@@ -111,6 +111,34 @@ internal void Register(Manifest mod)
 			Directory.CreateDirectory(supportPath);
 
 			File.WriteAllLines(Path.Combine(supportPath, key + ".yaml"), yaml.ToLines(false).ToArray());
+
+			// Clean up stale mod registrations:
+			//  - LaunchPath no longer exists (uninstalled)
+			//  - LaunchPath and mod match the current mod, but version differs (newer version installed on top)
+			List<string> toRemove = null;
```

Comment:
What is this for? I only see you adding things to it, but never actually doing something with those.